examples/3d/fog_volumes.rs

```diff
@@ -43,7 +44,9 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
             // up.
             scattering: 1.0,
             ..default()
-        });
+        })
+        // indicates that this fog volume needs to be Synchronized to the render world
+        .insert(SyncRenderWorld);
```

Comment:
Ouch this feels like a big footgun. I _really_ don't like that users have to be aware to add SyncRenderWorld for spatial entities that are parents of renderable-entities.

Nothing's immediately appearing in my brain as a solution, but I'd really like to brainstorm ways to avoid this.

Reply:
I suspect that this could be switched to a FogVolumeBundle, and that FogVolumeBundle should have SyncRenderWorld? If not, I do agree that this is a pretty major UX issu.